config/locales/en.yml

```diff
@@ -962,6 +962,10 @@ en:
       container_groups:            Container Groups
       container_replicator:        Replicator
       container_replicators:       Replicators
+      container_image:             Image
```

Comment:
Image can be a lot of things (search this file for images :)), so I suggest "container image"